frontend/view-edit-lead/edit-lead-inputs/lead-services-inputs.component.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import easyFetch from "../../util/easy-fetch";
+import { IntakeService } from "../../util/services-inputs.component";
+import IntakeServicesInputs from "../../util/services-inputs.component";
+
+export default function LeadServicesInformationInputs(
+  props: LeadServicesInformationInputsProps
+) {
+  const [services, setServices] = React.useState([]);
+  const [leadServices, setLeadServices] = React.useState(
+    props.lead.leadServices
+  );
+  const leadServicesRef = React.useRef(null);
+
+  React.useEffect(() => {
+    const abortController = new AbortController();
+
+    easyFetch("/api/services", { signal: abortController.signal }).then(data =>
+      setServices(data.services)
+    );
+
+    return () => abortController.abort();
+  }, []);
+
+  return (
+    <form onSubmit={handleSubmit} autoComplete="off" className="edit-form">
+      <label>
```

Comment:
Why does this `<label>` element exist? It doesn't seem like it would be helpful for accessibility, since there are many inputs inside of it, and even labels inside of it.